app/assets/stylesheets/views/_advanced-search.scss

```diff
@@ -26,3 +26,7 @@
   display: none;
 }
 // scss-lint:enable IdSelector
+
+.topic-finder__taxon-link {
```

Comment:
I think this needs removing? It's already in `finder_frontend.scss`.

Reply:
Good spot - it's been removed.